src/App.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import TaskBox from "./components/TaskBox";
+
+import BeadsLogo from "./logos/beads.svg";
+import MsitLogo from "./logos/msit.svg";
+import ProvocationLogo from "./logos/provocation.svg"
+
+function App() {
+  let query = new URLSearchParams(window.location.search);
+  const participantId = query.get("participantID");
+  const studyId = query.get("studyID");
+
+  return (
+    <div style={{ backgroundColor: "#41464a" }}>
+      <div className="container p-6" >
+        <h1 className="title is-1 is-flex is-justify-content-center has-text-white mb-6 pb-4">Tasks</h1>
+        <div className="columns">
+          {/* Column 1 for tasks */}
+          <div className="column is-half">
```

Comment:
(This is fully an optional thought to tinker with and not something you need to do if you don't want).  Instead of using `columns` and `is-half`, you could look into using css grid so that the design can be responsive: https://css-tricks.com/snippets/css/complete-guide-grid.  Then you can add a rule for `@mobile` in the scss that changes the grid columns  to full width. You'd also be able to pull out the logos/links to a config data structure and then just loop through the creation of TaskBoxes.  This could make this app super easy to reuse for some other study if needed.

Reply:
Done. One thing is that apparently using a variable in the import statement doesn't work, so import is set in the json file.